README.md

```diff
@@ -1,9 +1,30 @@
 jenkins-build-with-parameters-plugin
 ====================================
 
-Allows the user to provide parameters for a build in the url, prompting for confirmation before triggering the job.
+Allows the user to provide parameters for a build in the url, prompting
+for confirmation before triggering the job.
```

Comment:
Most of the time we prefer "sentence per line" so that it is easier to read the changes from one version to the next.  Not mandatory, but sentence per line makes comparisons easier (at least for me and a few others).
```suggestion
Allows the user to provide parameters for a build in the url, prompting for confirmation before triggering the job.
```

Reply:
okay @MarkEWaite will do the suggested changes